api/redisfailover/v1/validate.go

```diff
@@ -27,8 +27,17 @@ func (r *RedisFailover) Validate() error {
 		}
 		initialRedisCustomConfig = bootstrappingRedisCustomConfig
 	}
-
-	r.Spec.Redis.CustomConfig = append(initialRedisCustomConfig, r.Spec.Redis.CustomConfig...)
+	redisCustomConfigExists := false
+	for customConfig := range r.Spec.Redis.CustomConfig {
+		for initialConfig := range initialRedisCustomConfig {
+			if customConfig == initialConfig {
+				redisCustomConfigExists = true
```

Comment:
This looks like even if the one of the elements of `CustomConfig` is equal to `initialRedisCustomConfig`, then we will not append the `r.Spec.Redis.CustomConfig`!!! Am I right?